ChatApp/src/screens/SignUp_SignIn/SignIn_Screen.tsx

```diff
@@ -0,0 +1,157 @@
+import Styles from './Style';
+import React, { useEffect, useState } from 'react';
+import auth from '@react-native-firebase/auth';
+import { useNavigation } from '@react-navigation/native';
+import Antdesign from 'react-native-vector-icons/AntDesign';
+import SignIn_Button from '../../components/Buttons/SignIn';
+import { LoginManager, AccessToken } from 'react-native-fbsdk-next';
+import Social_Media_button from '../../components/Buttons/Social_Media';
+import { GoogleSignin } from '@react-native-google-signin/google-signin';
+import { View, Text, Image, TouchableOpacity, SafeAreaView } from 'react-native';
+
+const SignIn_Screen = () => {
+  const navigation = useNavigation();
+
+  const [userDetail, setUserDetail] = useState({});
+
+  console.log(userDetail);
+
+  useEffect(() => {
+    GoogleSignin.configure({
+      webClientId: '993010448494-junrsuvh8tq7g3ena1edbb8lnj43e0h5.apps.googleusercontent.com',
```

Comment:
We will have to get this from secret file

Reply:
Done